modules/ROOT/pages/constraints/syntax.adoc

```diff
@@ -314,9 +226,8 @@ SHOW [
   [WHERE expression]
 ----
 
-To get the full set of output columns, a yield clause is needed:
```

Comment:
feels odd to keep the description sentence for the WHERE case (```The simple version of the command allows for a `WHERE` clause and will give back the default set of output columns:```) but not the YIELD case 🤔